docs/docs/README.md

```diff
@@ -4,24 +4,72 @@ id: intro
 slug: /intro
 sidebar_position: 1
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { Gear, Plus } from "@phosphor-icons/react";
 
 
 ![Banner](/img/banner.png)
 
 ---
 
 > Welcome to Butterfly, the opensource note app.
-> Use the sidebar to navigate through the wiki.
+
+Butterfly is a note taking up that is aiming for unified experience across devices. Butterfly's approach is for minimalistic view of the drawing canvas while maximizing the user's control over the app.
```

Comment:
Umm How is it ?
```suggestion
Butterfly is a note taking app that is aiming for unified experience across all platforms. It provides simple UI with powerful customization.
```

Reply:
I'm fine with both, @CodeDoctorDE what do you think is better describing the app?

Reply:
Hmm I have no strong opinion on this. I think the minimalistic aspect is important but then you could fix the typo in `Butterfly is a note taking up`